### DISCUSSION TITLE
I'm not seeing the "Insert Note" function.

### DISCUSSION THREAD

**Discussion by GullyGully** [freeplane/freeplane#2066] [Help] [answered]
Upon right click of a node, I am not seeing the "insert note" function. Can someone please set me straight. Thanks.

**Comment by euu2021** [comment]
![image](https://github.com/user-attachments/assets/02adadca-2563-4c74-86b8-9d2bb584a84c)

**Reply by GullyGully** [reply, marked as answer]
Many thanks, one layer lower than expected!  From my perspective it would be nice to have quicker insert and edit access. But this works. Thanks again answering my question.

**Reply by i-plasm** [reply]
>  it would be nice to have quicker insert and edit access

This might help by allowing direct note edition: `View->Notes->Display note panel`. You can show it or hide it as needed.

**Reply by GullyGully** [reply]
Perfect. Thanks!
